CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,23 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
+func lottoNumbers () -> Set<Int>{
```

Comment:
```suggestion
func lottoNumbers() -> Set<Int> {
```

함수명은 보통 동사로 시작하는 편인 것 같습니다.